Dangerfile

```diff
@@ -0,0 +1,28 @@
+# By Jakub Kaspar 26/02/2020
+# MessageKit, 2020
+
+# This runs on CI
+mergeable_state = github.pr_json["mergeable_state"]
+
+# Make it more obvious that a PR a draft
+if mergeable_state == "draft"
+  warn("PR is marked as Draft")
+end
+
+# Mainly to encourage writing up some reasoning about the PR, rather than just leaving a title
+if github.pr_body.length < 5
+  fail("Please provide a summary in the Pull Request description")
+end
+
+if !git.modified_files.include?("CHANGELOG.md") && has_app_changes
+  fail("Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/MessageKit/MessageKit/blob/master/CHANGELOG.md).")
+end
+
+# Warn when there is a big PR
+if git.lines_of_code > 1000
```

Comment:
Can we make this 500? If it becomes a problem then we can loosen the value a bit

Reply:
It's both added + deleted + updated, so 1000 is not that many... I would keep it, because +-250 is really small amount when we enforce adding tests. :)